FlorenceBML/BMLor148.xml

```diff
@@ -38,6 +38,800 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           </desc>
+                        </item>
+                        <item xml:id="e8">
+                           <locus target="#7ra #151va #152rb #153rb #162va"/>
+                           <desc type="Correction">Small corrections to the Ethiopic text by a European hand.</desc>
+                        </item>
+                        <item xml:id="e9">
+                           <desc type="Correction">Corrections by an Ethiopian hand throughout the text.</desc>
+                        </item>
+                        <item xml:id="e10">
+                           <locus target="#203v"/>
+                           <desc type="StampExlibris">"207746" written on the bottom margin.</desc>
+                        </item>
+                        <item xml:id="e11">
+                           <locus target="#182r"/>
+                           <desc type="Unclear">A line of ill-shaped Syriac characters: 
+                              <foreign xml:lang="syr">ܢܡܐܫܦ ܐܜܚܒܔܩܚܣܝܬܢܬ</foreign></desc>
+                        </item>
+                        <item xml:id="e12">
+                           <desc type="StampExlibris">"N. 58 III" written on the top of the spine and partially faded.</desc>
+                        </item>
+                        <item xml:id="e13">
+                           <desc type="StampExlibris">"Orient" written in pencil on the bottom of the spine.</desc>
+                        </item>
+                        <item xml:id="e14">
+                           <desc type="StampExlibris">"148" written in ink on the bottom of the spine.</desc>
+                        </item>
+                     </list>
+                  </additions>
+                  <bindingDesc>
+                     <binding>
+                        <decoNote xml:id="b1" type="Boards">European binding in leather.</decoNote>    
+                        <decoNote xml:id="b2" type="SewingStations">4</decoNote>
+                        <decoNote xml:id="b3" type="bindingMaterial"><material key="wood"></material></decoNote>
+                     </binding>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate evidence="internal-date">Year <date when="1426">ወኮነ፡ ኍልቈ፡ ዓመተ፡ ዘቅድመ፡ አይኅ፡ 
+                        <hi rend="rubrig">፷፻፡ ወ፬፻፡ ወ፷፰፡</hi> ዓመታቲሁ፡</date> mentioned on <locus target="#181rb" corresp="#ms_i6"/>
+                        refers to <date when="1426"/>.</origDate>
+                  </origin>
+                  <provenance>
+                     <note>According to <bibl><ptr target="bm:Marrassini1987manoscrittietiopici"/><citedRange unit="page">92 and 95
+                     </citedRange></bibl>, most of the parts of the manuscript may have been copied to <ref type="mss" target="#BAVborget2"/>.
```

Comment:
I don't know both manuscripts well, but don't find Marrassini unambiguous. It might be that both manuscripts were copied from the same exemplar? He writes that the works were also copied, not precisely that it was this manuscript that was copied. I would maybe reformulate to state that the works in question are present in both manuscripts with common errors, and that the manuscripts were possibly produced in the same context.

Reply:
I don't know both manuscripts well, but don't find Marrassini unambiguous. It might be that both manuscripts were copied from the same exemplar? He writes that the works were also copied, not precisely that it was this manuscript that was copied. I would maybe reformulate to state that the works in question are present in both manuscripts with common errors, and that the manuscripts were possibly produced in the same context.

Reply:
Alright!